src/wrappers/index.ts

```diff
@@ -18,7 +18,6 @@
 
 export {
   AuthorizableWrapper,
```

Comment:
I hate how we have 2 index.js for `strategies` and `set_protocol`. Can we just consolidate into this file, so it doesn't need to be updated twice?